glossary.md

```diff
@@ -0,0 +1,7 @@
+You may also need to check https://chromium.googlesource.com/chromiumos/docs/+/master/glossary.md.
+
+* LGTM: "Looks good to me", commonly used to approve a code review.
+* RSLGTM: "Rubber stamp looks good to me", used when the reviewer is merely granting OWNERS approval without doing a proper code review.
+* WIP: "Work In Progress" - e.g. a patch that's not finished, but may be worth an early look
+* WPT: [web-platform-tests](https://github.com/web-platform-tests/wpt)
+* godbolt: [Compiler Explorer](https://godbolt.org/) was created by and is primarily administrated by Matt Godbolt.
```

Comment:
I'm not sure if the information about the author is necessary. I would rather have more information about the tools capability.

Reply:
```suggestion
* godbolt: [Compiler Explorer](https://godbolt.org/) run compilers interactively from your web browser and interact with the assembly. Was created by and is primarily administrated by Matt Godbolt.
```

Reply:
Thanks, updated.